src/components/ToolMenu/Draw/StylingDrawerButton/index.tsx

```diff
@@ -24,9 +25,14 @@ export const StylingButton: React.FC = (): JSX.Element => {
   const {
     t
   } = useTranslation();
+  const drawerVisibility = useAppSelector(state => state.stylingDrawerVisibility);
 
   const showDrawer = () => {
```

Comment:
since this shows and hides the drawer the name does not fit. maybe change this to toggleDrawer?